### ISSUE TITLE
Unable to decode base4 encoded fields in json message

### ISSUE THREAD

**Issue by lfpcorreia** [tchiotludo/akhq#653]
Hi! First of all I can't say enough how useful this tool is! :) 

I'm assuming this is possible:

given a message like

``` json
{
"elem1": "lorem ipsum",
"elem2": 1234,
"elem3": "AXiHvYoSWHFc4NOr1rbZGQ=="
}
```
something like this:
``` toml
deserialization:
        protobuf:
          topics-mapping:
            - topic-regex: "sample-topic"
              descriptor-file: "elem3"
```
would decode the base64 encoded `elem3` in the message.

Is it possible?

**Comment by tchiotludo** [comment]
Hi :) 
I'm not sure to understand here :) 
You base message are in protobuf ?

**Comment by lfpcorreia** [comment]
Nope, my messages are in _plain-old-good_ _human-readable_ json format :) 

But I have field(s) that are base64 encoded for several reasons.

I want to be able to decode those specific fields.

Any change I can make it work?

**Comment by tchiotludo** [comment]
It will need a manual mapping from topic to rules to in order to decode this.
This is a new feature not available for now. 
But PR are welcome !
